### ISSUE TITLE
error "javax.validation.constraints.Past.message" 

### ISSUE THREAD

**Issue by mobidevadmin** [Graylog2/graylog2-server#2002]
### Problem description
We are receiving such messages in logs quite often
### Steps to reproduce the problem
There are no exact steps, it's been from the very first installation. W've double checked the date settings on all graylog-collector nodes. Tried to wipe mongodb data, upgraded graylog-server and java. Also, shut down all graylog-collectors and brought them up one by one and message occurs randomly.
Message sample:

```
2016-03-30T19:33:30.851+03:00 ERROR [AnyExceptionClassMapper] Unhandled exception in REST resource
java.lang.IllegalArgumentException: Specified object failed validation: [ConstraintViolationImpl{interpolatedMessage='must be in the past', propertyPath=lastSeen, rootBea
nClass=class org.graylog2.collectors.AutoValue_CollectorImpl, messageTemplate='{javax.validation.constraints.Past.message}'}]
        at org.graylog2.collectors.CollectorServiceImpl.save(CollectorServiceImpl.java:68)
        at org.graylog2.rest.resources.system.collector.CollectorResource.register(CollectorResource.java:117)
        at sun.reflect.GeneratedMethodAccessor69.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory$1.invoke(ResourceMethodInvocationHandlerFactory.java:81)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:144)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:161)
        at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:160)
        at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:99)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:389)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:347)
        at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:102)
        at org.glassfish.jersey.server.ServerRuntime$2.run(ServerRuntime.java:326)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:271)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:267)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:315)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:297)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:267)
        at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:317)
        at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:305)
        at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:1154)
        at org.graylog2.jersey.container.netty.NettyContainer.messageReceived(NettyContainer.java:356)
        at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
        at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
        at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
        at org.jboss.netty.handler.execution.ChannelUpstreamEventRunnable.doRun(ChannelUpstreamEventRunnable.java:43)
        at org.jboss.netty.handler.execution.ChannelEventRunnable.run(ChannelEventRunnable.java:67)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:176)
        at org.jboss.netty.handler.execution.MemoryAwareThreadPoolExecutor$MemoryAwareRunnable.run(MemoryAwareThreadPoolExecutor.java:606)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
```

### Environment

* Graylog Version:
1.3.4-1
* Elasticsearch Version:
1.7.5
* MongoDB Version:
1:2.4.9-1ubuntu2
* Operating System:
Ubuntu 14.04.4 LTS
* Browser version:
firefox v.45

**Comment by kroepke** [comment]
Same as https://github.com/Graylog2/graylog2-web-interface/issues/1726

**Comment by kroepke** [comment]
It looks like a race condition to me.
What happens is this:
When a collector sends its ping, the server creates a timestamp with the current time (in milliseconds).
It then tries to create a database entry, which during save triggers a validation of the object.
The last seen timestamp is declared as having to be in the past.
That check is implemented by comparing the current time to the timestamp in the object, but the comparison is done with a `>` and on the milliseconds.
If the entire operation happens during the same millisecond, the timestamp is not in the "past" but in the "present".
I believe we can safely skip the validation of that field, because the server is generating the timestamp and not any client or collector.

**Comment by bernd** [comment]
This will be fixed in the  upcoming Graylog version 2.0. (>= beta.2)
